README.md

```diff
@@ -9,6 +9,49 @@ This started as a fork of [Play-ReactiveMongo](https://github.com/ReactiveMongo/
 With some minimal effort, as the ReactiveMongo people had already done the majority of the work, we felt that adding a base repository class creates a library without some
 of the issues the other simpler libraries have.
 
+## Upgrading from 5.x.x to 6.x.x?
+
+With version 6.x.x of simple-reactivemongo, we are moving to the latest version of reactivemongo which comes with a few braking changes documented here:  
+http://reactivemongo.org/releases/0.12/documentation/release-details.html#breaking-changes
+
+You will most likely encounter some of the following issues. Please have a look on how we recommend to fix them.
+
+#### No Json serializer as JsObject found 
+
+Due to keeping parity with upstream, companion objects for inner classes do not have the ImplicitBSONHandlers handlers.
+If you see an error like this:
+```No Json serializer as JsObject found for type reactivemongo.bson.BSONDocument. Try to implement an implicit OWrites or OFormat for this type.```
+Try adding the following import
+```scala
+import reactivemongo.play.json.ImplicitBSONHandlers._
+```
+
+#### WriteResult is no longer an Exception
+
+The type hierarchy of the trait `WriteResult` has changed in new version of reactivemongo. 
+It’s no longer an Exception. As it now longer represents errors in the public API, the following properties have been removed: `errmsg`, `hasErrors`, `inError` and `message`.
+
+simple-reactivemongo previously exposed `inError`, something that is no longer possible.
+
+Also, given that the `LastError` was part of the `WriteResult`, it used to be returned in a `Future.Success`, now it's returned in a `Future.Failure`.
+
+If you have something like this in your code:
+```scala
+collection.doSomething([...]) map {
+    case lastError if lastError.inError => // handle the error
+    case _ => // return successfully
+}
+```
+
+Now, you have to do this:
+```scala
+collection.doSomething([...]) map {
+    case _ => // return successfully
+} recover {
+   case lastError => // handle the error
```

Comment:
Might be worth changing this to:

```
collection.doSomething([...]) map {
     case _ => // return successfully
 } recover {
    case lastError: LastError => // Handle DB specific issue
    case e => // Handle other exceptions
}
```

Reply:
The example of the old version wasn't. I prefer to keep focus on the use of lastError

Reply:
Ok, either way the main point is given across.